wp-content/plugins/core/src/Theme/Theme_Definer.php

```diff
@@ -106,25 +106,39 @@ public function define(): array {
 				] ),
 
 			/**
-			 * The TypeKit kit ID
+			 * @var string A valid TypeKit (Adobe Fonts) Project ID.
+			 *
+			 * Example: `zbo7iia`
 			 */
 			self::CONFIG_TYPEKIT_ID => '',
 
 			/**
-			 * Define the Google font families to load
+			 * @var array A collection of Google Font families to load.
+			 *
+			 * Any combination of valid Google font family declarations is acceptable.
+			 *
+			 * Example: `[ 'Tangerine', 'Cantarell:italic', 'Droid+Serif:b' ]`
+			 *
+			 * @link https://developers.google.com/fonts/docs/getting_started
 			 */
 			self::CONFIG_GOOGLE_FONTS => [],
 
 			/**
-			 * Define the custom font families to load
+			 * @var string CSS file URL.
+			 *
+			 * A complete URL to a css file containing custom @font-face definitions.
+			 * Useful for other 3rd-party web font providers such as fonts.com.
+			 *
+			 * Example: `'https://fast.fonts.net/cssapi/e6dc9b99-64fe-4292-ad98-6974f93cd2a2.css'`
+			 *
 			 */
-			self::CONFIG_CUSTOM_FONTS => [],
+			self::CONFIG_CUSTOM_FONTS => '',
```

Comment:
Is it possible that we might have multiple sources for custom fonts? I.e., should this be an array of URLs, or just a single string?

Reply:
Good question, I wasn't planning on multiple; but after discussion w/ FE, we decided it's not a bad idea to make it support multiple. This has been updated.